src/sass/_Fabric.Icons.scss

```diff
@@ -347,3 +347,20 @@
     z-index: $ms-zIndex-back;
   }
 }
+
+// Fabric Core Icon variables
+$ms-Icon-12: 12px;
+$ms-Icon-16: 16px;
+$ms-Icon-20: 20px;
```

Comment:
I'm leaning in this direction as well. The benefit of using 12, 16, and 20 is that if we were to introduce a new size of 18 it would fit right in. The downside is that we're not matching the convention of our other styles, and we're not make even an implicit suggestion of which is the base size. Let's use small, medium, and large and if we do need to add another size in between, we can adjust the ramp to reflect the typical usage.